infrastructure/jest.config.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
   testMatch: ['**/*.test.ts'],
   transform: {
     '^.+\\.tsx?$': 'ts-jest'
-  }
+  },
+  collectCoverage: true,
```

Comment:
Note: This starts collecting coverage during an `npm run test` but this _does not_ add SonarCloud coverge checking for the `infrastructure/` directory yet (and that's intentional).